pwem/convert/symmetry.py

```diff
@@ -27,11 +27,11 @@
 This module returns the matrices related with the different
 point symmetries. Code based on chiomera file sym.py
 """
-from pyworkflow.em.constants import SYM_CYCLIC, SYM_DIHEDRAL, SYM_OCTAHEDRAL, \
-    SYM_TETRAHEDRAL, SYM_TETRAHEDRAL_Z3, SYM_I222, SYM_I222r, \
-    SYM_In25, SYM_In25r
+from pwem.constants import (SYM_CYCLIC, SYM_DIHEDRAL, SYM_OCTAHEDRAL,
```

Comment:
Maybe we should agree in a prefix and use. I find this kind of imports no so clean.
Moreover, it is good to know in the code where things come from.